addons/mail/static/tests/mock_server/mail_mock_server.js

```diff
@@ -743,6 +743,83 @@ async function mail_message_update_content(request) {
     ).get_result();
 }
 
+registerRoute("/mail/message/get_followers", message_get_followers);
+/** @type {RouteCallback} */
+async function message_get_followers(request) {
+    const {
+        thread_id,
+        thread_model,
+        limit,
+        offset,
+        filter_recipients = false,
+    } = await parseRequestParams(request);
+    if (filter_recipients) {
+        // not implemented for the simplicity
+    }
+
```

Comment:
If that doesn't cost much, would implement the `filter_recipients`.
We know maintaining mock server sucks, we'll probably do something eventually for that.
But in the meantime let's try as much as possible to match python implementation when equivalent methods are implemented. We should expect that when a controller / python method is defined in JS mock models, this is (mostly) covered. It's ok to not cover the stuffs that we don't care for UI/UX like mail transport, but `filter_recipients` affects returned results so should be adapted.